lib/rtc_peer.js

```diff
@@ -221,17 +226,12 @@ export class RTCPeer extends EventEmitter {
             // to be 'sendrecv' so Firefox stops complaining.
             // In practice the transceiver is only ever going to be used to
             // receive.
-            for (const t of this.pc.getTransceivers()) {
-                if (t.receiver && t.receiver.track === ev.track) {
-                    if (t.direction !== 'sendrecv') {
-                        this.logger.logDebug('RTCPeer.onTrack: setting transceiver direction for track');
-                        t.direction = 'sendrecv';
-                    }
-                    break;
-                }
+            if (ev.transceiver.direction !== 'sendrecv') {
+                this.logger.logDebug('RTCPeer.onTrack: setting transceiver direction for track');
+                ev.transceiver.direction = 'sendrecv';
```

Comment:
No functional change here. I just realized that the transceiver is included in the event so we don't have to find it.